server/src/main/java/org/spine3/server/command/error/CommandExpiredException.java

```diff
@@ -32,6 +32,8 @@
 /**
  * Exception that is thrown when a scheduled command was not delivered to the target in time.
  *
+ * <p>For example, in the case when a server restarted when it was the time to deliver a scheduled command.
```

Comment:
Why don't we deliver it after when the server is started back again? What is the standard practice for such cases in messaging systems?